### DISCUSSION TITLE
How to support Chinese Search Segmentation?

### DISCUSSION THREAD

**Discussion by bp404** [web-infra-dev/rspress#1084] [Q&A] [answered]
For example, searching for "环境[space or other separators]部署" can be matched to "环境的离线部署".

**Comment by Timeless0911** [comment, marked as answer]
The current local search is based on flexsearch. 

You can try to optimize the matching of fuzzy search in this file https://github.com/web-infra-dev/rspress/blob/75fbb8e11dbfba2127d3f0b28ac9a4cf6c725e88/packages/theme-default/src/components/Search/logic/providers/LocalProvider.ts

**Comment by bp404** [comment]
Got it. Thanks
